src/intra_edge.rs

```diff
@@ -18,18 +18,22 @@ bitflags! {
 }
 
 impl EdgeFlags {
-    const LEFT_HAS_BOTTOM: Self = Self::union_all([
+    pub const LEFT_HAS_BOTTOM: Self = Self::union_all([
```

Comment:
```suggestion
    pub const ALL_LEFT_HAS_BOTTOM: Self = Self::union_all([
```

I should've done this when I added this, but could add this in a separate commit on this PR?